specs/commandline/blob.md

```diff
@@ -139,6 +140,20 @@ Flags:
   -h, --help      help for import
 ```
 
+### notation blob policy init
+
+```text
+Initialize blob trust policy configuration
+
+Usage:
+  notation blob policy init --name <policy_name> --trust-store "<store_type>:<store_name>" --trust-identity "<trust_identity>"
```

Comment:
```suggestion
  notation blob policy init --name <policy_name> --trust-store "<store_type>:<store_name>" --trusted-identity "<trusted_identity>"
```
Should it be `trusted identity`? https://github.com/notaryproject/specifications/blob/main/specs/trust-store-trust-policy.md#trust-policy